.github/workflows/run-tutorial-ue.yml

```diff
@@ -0,0 +1,36 @@
+name: run-tutorial-ue
+on:
+  push: 
+    branches:
+      - main
+  pull_request:

```

Comment:
would ensure this only runs on changes to the tutorial's dir, see other workflow files

Reply:
These changes are made in https://github.com/Azure/azureml-examples/pull/199